### PR TITLE
Azure integration

### PR DESCRIPTION
**Pull request by mbenguig**

### CODE REVIEW

File: endpoints/2-cloud-endpoints.md [hunk 1]
```diff
@@ -88,10 +87,11 @@ Note that cloud credentials are validated only during async process.
     "cloudProviderName": "azure",
     "cloudType": "PUBLIC",
     "subnet": null,
-    "securityGroup": "{{azure-securityGroup}}",
+    "securityGroup": null,
     "sshCredentials": {
-      "username": "{{azure-username}}",
+      "username": "ubuntu",
```
Comment (ankicabarisic) [inline endpoints/2-cloud-endpoints.md:92]: i would keep it here as it was as this is requirement for the nebulous scripts

**Review by ankicabarisic — APPROVED** [review]
just one small comment

File: endpoints/2-cloud-endpoints.md [hunk 2]
```diff
@@ -134,7 +134,8 @@ Contains SSH access information for the cloud. For Open Stack and AWS should be
 
     - `username` (string): The SSH username.
     - `keyPairName` (string): The name of the key pair used for SSH access.
-    - `privateKey` (string or `null`): The private key in RSA format, with line breaks represented by `\n` for JSON compatibility. If not required, use `null`.
+    - `publicKey` (string or `null`): The public key in RSA format. If not required, use `null`.
```
Comment (ankicabarisic) [inline endpoints/2-cloud-endpoints.md:137]: do we have publicKey for add cloud. i think it is just private one.

File: endpoints/2-cloud-endpoints.md (hunk 1 above)
Comment (mbenguig) [reply endpoints/2-cloud-endpoints.md:92]: After our discussion, let's leave it like this as a workaround of the unknown ubuntu user issue

File: endpoints/2-cloud-endpoints.md (hunk 2 above)
Comment (mbenguig) [reply endpoints/2-cloud-endpoints.md:137]: Thx to this PR .. yes!

**Pull request merged**